src/module.ts

```diff
@@ -220,6 +220,12 @@ export interface ModuleOptions {
     injectPage?: boolean
     trailingSlash?: boolean
   },
+  /**
+   * Enable to keep uppercase characters in the generated routes.
+   *
+   * @default false
+   */
+  keepUppercase: boolean
```

Comment:
This name may be too vague since we do not understand it's related to path. 

Maybe something like respectPathCase, when false, it lower case or respectFilenameCase

Reply:
I've rename the name to `respectPathCase`.